### PR TITLE
Hello World in Moonscript

### PR DESCRIPTION
**Pull request by BassemMohamed**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [x] I added fun facts (i.e. debut, developer, typing, etc.)
  - [x] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [x] I fixed #375 

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

Hey, this is my first pull request ever (Super Happy) !!

### CODE REVIEW

File: archive/m/moonscript/hello-world.moon [hunk 1]
```diff
@@ -0,0 +1 @@
+print "Hello World!"
```
Comment (jrg94) [inline archive/m/moonscript/hello-world.moon]: Any chance you can update the string for consistency? `"Hello, World!"`

Comment (jrg94) [reply archive/m/moonscript/hello-world.moon]: Also, can you add a newline to the end of this file?

File: archive/m/moonscript/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Sample Programs in MOONSCRIPT
```
Comment (chrboe) [inline archive/m/moonscript/README.md]: I'm not 100% sure, but I think the official spelling is "MoonScript". Could you clarify this? imo it's better to consistently use the official capitalization

Comment (BassemMohamed) [reply archive/m/moonscript/README.md]: yeah thats correct!

**Comment by chrboe** [comment]
Looks like I can't resolve @jrg94 s review comments, so we'll have to wait for him to approve before merging.

**Comment by jrg94** [comment]
Haha that's very bizarre, but it makes sense I suppose. Anyway! All done.

**Pull request merged**